Dockerfile

```diff
@@ -1,4 +1,5 @@
 ARG IMAGE_REF=latest
+ARG ARCH=x86_64
```

Comment:
This line needs to be moved somewhere below `FROM` line. Any ARG specified above `FROM` is used only for `FROM` line and not in later steps.